api/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class DeliveryStatus(models.TextChoices):
+    UNKNOWN_ERROR = "UNKNOWN_ERROR", _(u"Error desconegut")
+    ERROR_FROM_BODY = "ERROR_FROM_BODY", _(u"Error alta enviament")
+    LABEL_SENT = "LABEL_SENT", _(u"Etiqueta enviada al venedor")
+
+
+class Shipment(models.Model):
+    shipment = models.CharField(max_length=100, null=False, blank=False)
+    order = models.IntegerField(null=False, blank=False)
+    vendor = models.IntegerField(null=False, blank=False)
+    label_url = models.CharField(max_length=200, blank=True)
+
+    status = models.CharField(
+        _(u"Estat de l'enviament"),
+        max_length=20,
+        choices=DeliveryStatus.choices,
+        null=False,
+        blank=False,
+    )
+
+    def __str__(self):
+        return self.shipment
```

Comment:
why do we need this?

Reply:
Flake8 was complaining about not having it declared.

./api/models.py:11:1: DJ08 Model does not define __str__ method

Reply:
it will also help in the django admin visualization to be more explicit than just the ID of the db table.

Reply:
I've add a test line to cover this. Check: https://github.com/coopdevs/lazona_connector/pull/87/commits/a3e9a2a879597490c8e697e8ff3284d24840107a